finance/jobs/2020_githubresponder.rst

```diff
@@ -0,0 +1,25 @@
+GitHub First/Last Responder
+===========================
+
+The Astropy Project is looking to hire a Github first responder / last responder. The overarching goal of this position is to improve the first time contributor experience, reduce the technical barriers to contributions, reduce the number of open issues, and increase the number of merged pull requests. This is not intended to cover all issues and PRs opened in astropy, but rather to support and lighten the load on the rest of the volunteer community.
+An ideal candidate would be familiar with the astropy project ecosystem and capabilities, comfortable with the Github Development workflow, and have experience providing supportive feedback and technical advice in online environments.
```

Comment:
Astropy's workflow is not just any dev workflow. For instance, someone familiar with Gitflow might not be familiar with the variant that we use, so better be a little more specific.
```suggestion
An ideal candidate would be familiar with the Astropy project ecosystem and capabilities, comfortable with its Github Development workflow, and has experience providing supportive feedback and technical advice in online environments.
```